### DISCUSSION TITLE
How to prevent any uploads to the cloud?

### DISCUSSION THREAD

**Discussion by ThePlenkov** [nrwl/nx#11617] [General]
Hi!

Following CI/CD docs I've set up a pipeline. Now I see that nx affected tasks upload results to the cloud.

Do you know - is it possible to avoid nx uploading any data to public cloud? Is there any global switch? How many commands are affected? I feel not safe that running something locally makes it public without even informing me.

Thank you!

**Comment by ThePlenkov** [comment]
Ok I have found the answer: i had to use `nx/tasks-runners/default` default task runner instead of `@nrwl/nx-cloud`

**Reply by DheerajMahra** [reply]
This doesn't work for me in module fedeartion setup using nx, one of the remote app breaks
